plugins/GSdx/GSdx.rc

```diff
@@ -154,24 +154,26 @@ BEGIN
-    RTEXT           "Skipdraw:",IDC_STATIC,44,174,32,8
-    EDITTEXT        IDC_SKIPDRAWHACKEDIT,80,171,116,14,ES_RIGHT | ES_AUTOHSCROLL
-    CONTROL         "",IDC_SKIPDRAWHACK,"msctls_updown32",UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,105,169,11,14
-    RTEXT           "TC Offset X:",IDC_STATIC,32,189,44,8
-    EDITTEXT        IDC_TCOFFSETX2,80,186,116,14,ES_RIGHT | ES_AUTOHSCROLL
-    CONTROL         "",IDC_TCOFFSETX,"msctls_updown32",UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,127,184,11,14
-    RTEXT           "TC Offset Y:",IDC_STATIC,32,204,44,8
-    EDITTEXT        IDC_TCOFFSETY2,80,201,116,14,ES_RIGHT | ES_AUTOHSCROLL
-    CONTROL         "",IDC_TCOFFSETY,"msctls_updown32",UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,127,199,11,14
+    RTEXT           "Skipdraw Range:",IDC_STATIC,14,174,62,8
+    EDITTEXT        IDC_SKIPDRAWOFFSETEDIT,80,171,58,14,ES_RIGHT | ES_AUTOHSCROLL | ES_NUMBER
+    CONTROL         "",IDC_SKIPDRAWOFFSET,"msctls_updown32",UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,0,0,11,14
+    EDITTEXT        IDC_SKIPDRAWHACKEDIT,138,171,58,14,ES_RIGHT | ES_AUTOHSCROLL | ES_NUMBER
+    CONTROL         "",IDC_SKIPDRAWHACK,"msctls_updown32",UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,0,0,11,14
+    RTEXT           "TC Offset X:",IDC_STATIC,32,190,44,8
+    EDITTEXT        IDC_TCOFFSETX2,80,187,116,14,ES_RIGHT | ES_AUTOHSCROLL | ES_NUMBER
+    CONTROL         "",IDC_TCOFFSETX,"msctls_updown32",UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,127,185,11,14
+    RTEXT           "TC Offset Y:",IDC_STATIC,32,206,44,8
+    EDITTEXT        IDC_TCOFFSETY2,80,203,116,14,ES_RIGHT | ES_AUTOHSCROLL | ES_NUMBER
+    CONTROL         "",IDC_TCOFFSETY,"msctls_updown32",UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,127,201,11,14
     // OpenGL Advanced Settings:
-    GROUPBOX        "OpenGL Advanced Settings",IDC_STATIC,7,225,196,48,0,WS_EX_TRANSPARENT
-    RTEXT           "Geometry Shader:",IDC_GEOMETRY_SHADER_TEXT,14,240,62,8
-    COMBOBOX        IDC_GEOMETRY_SHADER_OVERRIDE,80,237,97,63,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    RTEXT           "Image Load Store:",IDC_IMAGE_LOAD_STORE_TEXT,14,255,62,8
-    COMBOBOX        IDC_IMAGE_LOAD_STORE,80,252,97,63,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    GROUPBOX        "OpenGL Advanced Settings",IDC_STATIC,7,226,196,48,0,WS_EX_TRANSPARENT
+    RTEXT           "Geometry Shader:",IDC_GEOMETRY_SHADER_TEXT,14,241,62,8
+    COMBOBOX        IDC_GEOMETRY_SHADER_OVERRIDE,80,238,97,63,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    RTEXT           "Image Load Store:",IDC_IMAGE_LOAD_STORE_TEXT,14,256,62,8
+    COMBOBOX        IDC_IMAGE_LOAD_STORE,80,253,97,63,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
```

Comment:
Since you're changing gl hacks maybe you can adjust IDC_GEOMETRY_SHADER_OVERRIDE, IDC_IMAGE_LOAD_STORE width to match HPO, Sprite ..etc